database/insertData/dev/07_review_assignments.js

```diff
@@ -435,6 +435,7 @@ exports.queries = [
           stageNumber: 2
           levelNumber: 2
           isLastLevel: true
+          isSelfAssignable: true
```

Comment:
Do you think we still need to bother maintaining these? 

There's a few template features in Feature Showcase I think that could do with being pulled into the main snapshot, but apart from that, we may as well deprecate these scripts, do you think?

Reply:
Yes, I think that should be opk to do. I was only afraid we could need to start from scratch again. But I think that keeping previous versions of snapshots is safe. So yes, I can just delete these folders previously used to insert using js files.

Reply:
Ok - in order to keep `insertData.ts` file (which can be used as part of `snapshotCLI` insert) I had to keep the folder `insertData/_common` in, so deleting all of the others...

Reply:
I'm not sure we want to delete them all quite yet, just that we don't need to keep maintaining them. I think there's some stuff in Feature Showcase I'd like to get at some point, so good to have that still around.

Reply:
Oh.... ok. Reverting